xcode/Subconscious/Shared/Components/AddressBook/AddFriendViaQRCodeView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  AddFriendViaQRCodeView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 6/3/2023.
+//
+
+import Foundation
+import SwiftUI
+import CodeScanner
+import os
+
+struct AddFriendViaQRCodeView: View {
+    @Environment(\.dismiss) var dismiss
+    
+    var onScannedDid: (String) -> Void
+    var onCancel: () -> Void
+    
+    static let logger = Logger(
+        subsystem: Config.default.rdns,
+        category: "AddFriendViaQRCodeView"
+    )
+    
+    var body: some View {
+        VStack {
+            // Toolbar is unavailable here, create a minimal one that blends with notch/bezel.
+            HStack {
+                Button("Cancel", role: .cancel) {
+                    dismiss()
+                }
+                .buttonStyle(.borderless)
+
+                Spacer()
+            }
+            .padding(AppTheme.unit2)
+            .background(Color.black)
+            
+            CodeScannerView(
+                codeTypes: [.qr],
+                showViewfinder: true,
+                simulatedData: "did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7",
```

Comment:
Returned in the simulator when testing, since we can't access the camera in that context.

Reply:
I extracted this to a property in `Config` that only returns data when `debug` is true, feels like this view shouldn't care about the value itself and should remain dumb.